### PR TITLE
Limbus Labs Additions

### PR DESCRIPTION
**Pull request by tile4**
## About The Pull Request
This adds a few things and makes a few changes for abnormalities to make them more RP friendly

- Added Eris, Scorched Girl, Child of the Galaxy, Schad, Woodsman, and General Bee
- Makes Eris leave the brain upon feasting on a body in LCL
- Adds an ability to KoD, Monk, Piscine, Scorched, Murderer and General Bee to toggle their sprites to their contained sprites for LCL only.
- Adds an ability for LCL only for Galaxy Child to give gifts and break then, as well as KoD to give her gift
- Makes Monk's panting not play in LCL.
- Add a note to security officers that they are free to talk with abnos when they aren't busy.
- Adds Torso Fabricator to LCL with way cheaper prices
- General Bee spawns with 2 Bee soldiers
- Janitor and Archivist no longer take fear damage.
- Abnromalities are no longer hostile when unpossessed. Peccatulum still are though.
- Nothing There is no longer invincible
- Security Tape now breaks with less damage.
- You can drag your ghost to possess non-abnormalities in LCL

## Why It's Good For The Game
More RP stuff is nice for the RP mode.

## Changelog
:cl:
add: Added new things
tweak: tweaked a few things
balance: rebalanced something
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/he/galaxy_child.dm [hunk 1]
```diff
@@ -37,6 +38,86 @@
...
+		var/mob/living/carbon/human/new_friend = input(src, "Choose who you want to gift a pebble to", "Select your new friend") as null|anything in nearby // pick someone from the list
+		new_friend.apply_status_effect(STATUS_EFFECT_FRIENDSHIP)
+		galaxy_friend |= new_friend
+		heal_amount += heal_mod
+		damage_amount += damage_mod
+		RegisterSignal(new_friend, COMSIG_LIVING_DEATH, PROC_REF(FriendDeath))
+		icon_state = "galaxy"
+		depressed = FALSE
+
+datum/action/cooldown/galaxygiftbreak
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/he/galaxy_child.dm:85]: yeah uh... this is not good
```suggestion
/datum/action/cooldown/galaxygiftbreak
```

File: code/modules/mob/living/simple_animal/abnormality/he/pisc_mermaid.dm [hunk 2]
```diff
@@ -59,6 +59,37 @@
 	var/obj/item/clothing/head/unrequited_crown/crown
 	var/mob/living/carbon/human/love_target
 
+	attack_action_types = list(
+		/datum/action/innate/change_icon_merm
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/he/pisc_mermaid.dm:63]: preferably if you are doing multi-line lists, always have an object end with a comma. It makes future additions look better on github
```suggestion
		/datum/action/innate/change_icon_merm,
```

File: code/modules/mob/living/simple_animal/abnormality/teth/scorched_girl.dm [hunk 3]
```diff
@@ -40,6 +40,29 @@
 	var/boom_damage = 250
 	patrol_cooldown_time = 10 SECONDS //Scorched be zooming
 
+	attack_action_types = list(
+		/datum/action/innate/change_icon_scorch
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/teth/scorched_girl.dm:44]: preferably if you are doing multi-line lists, always have an object end with a comma. It makes future additions look better on github
```suggestion
		/datum/action/innate/change_icon_scorch,
```

File: code/modules/mob/living/simple_animal/abnormality/waw/clouded_monk.dm [hunk 4]
```diff
@@ -56,7 +56,7 @@
 	var/damage_taken
 
 	attack_action_types = list(
-		/datum/action/innate/abnormality_attack/toggle/monk_charge
+		/datum/action/innate/abnormality_attack/toggle/monk_charge, /datum/action/innate/change_icon_monk
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/waw/clouded_monk.dm:59]: please just add another line to this, it looks much better
```suggestion
		/datum/action/innate/abnormality_attack/toggle/monk_charge,
		/datum/action/innate/change_icon_monk,
```

File: code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm [hunk 5]
```diff
@@ -52,6 +53,70 @@
 	var/teleport_cooldown_time = 20 SECONDS
 	var/swords = 0
 
+	attack_action_types = list(
+		/datum/action/innate/change_icon_kod, /datum/action/cooldown/knightblessing
```
Comment (Gboster-0) [inline code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm:57]:
```suggestion
		/datum/action/innate/change_icon_kod,
		/datum/action/cooldown/knightblessing,
```

**Review by Gboster-0 — COMMENTED** [review]
mostly nitpicks, one really major thing

File: ModularTegustation/tegu_items/limbus_labs/!abno_overwrites.dm [hunk 6]
```diff
@@ -9,3 +9,18 @@
...
+		boom_damage = 80
+
+/mob/living/simple_animal/hostile/abnormality/general_b/Initialize()
+	..()
+	if(SSmaptype.maptype == "limbus_labs")
+		health = 1000
+		maxHealth = 1000
+		damage_coeff = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 1)
+		melee_damage_lower = 20
+		melee_damage_upper = 32
```
Comment (LanceSmites328) [inline ModularTegustation/tegu_items/limbus_labs/!abno_overwrites.dm:26]:
```suggestion
		melee_damage_upper = 32

```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm [hunk 7]
```diff
@@ -229,6 +259,22 @@
 	attack_sound = 'sound/weapons/bite.ogg'
 	speak_emote = list("buzzes")
 
+/mob/living/simple_animal/hostile/soldier_bee/Initialize()
+	. = ..()
+	if(SSmaptype.maptype == "limbus_labs")
+		faction = list("neutral")
+
+mob/living/simple_animal/hostile/soldier_bee/Login()
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm:267]:
```suggestion
/mob/living/simple_animal/hostile/soldier_bee/Login()
```

**Review by LanceSmites328 — DISMISSED** [review]

**Comment by Kitsunemitsu** [comment]
Please update the changelog for a quick checklist

**Comment by tile4** [comment]
I think that should be good.

**Review by Kitsunemitsu — APPROVED** [review]
Everything here looks good to me. 
Approved.

**Pull request merged**